src/confuse/multi_class_metrics.clj

```diff
@@ -67,3 +67,40 @@
   (let [tpisum (apply + (mapv (partial b/true-positives actual predicted) classes))
         fnisum (apply + (mapv (partial b/false-negatives actual predicted) classes))]
     (double (/ tpisum (+ tpisum fnisum))))))
+
+
+
+(defn- t [cm k]
```

Comment:
a more descriptive name would be nice, perhaps `row-sum` ? Similarly `column-sum` for line 76?

Reply:
they come from the MCC formula in the wikipedia, maybe they should become local functions.
It is the first time I converted some maths to Clojure, so I  am open to best practices...